editor/editor_properties.cpp

```diff
@@ -3570,6 +3571,45 @@ static EditorPropertyRangeHint _parse_range_hint(PropertyHint p_hint, const Stri
 	return hint;
 }
 
+static EditorProperty *get_input_action_editor(const String &p_hint_text, bool is_string_name) {
+	// TODO: Should probably use a better editor GUI with a search bar.
+	//Said GUI could also handle showing builtin options, requiring 1 less hint.
+	EditorPropertyTextEnum *editor = memnew(EditorPropertyTextEnum);
+	Vector<String> options;
+	Vector<String> builtin_options;
+	List<PropertyInfo> pinfo;
+	ProjectSettings::get_singleton()->get_property_list(&pinfo);
+	Vector<String> hints = p_hint_text.remove_char(' ').split(",", false);
+
+	//Code that shows an error when an invalid hint is entered
+	//for (const String& hint : hints) {
+	//    if (hint != "show_builtin" || hint != "loose_mode") {
+	//        ERR_FAIL_V_MSG(get_input_action_editor("", is_string_name), vformat("Invalid PROPERTY_HINT_INPUT_NAME with hint \"%s\"", hint));
+	//    }
+	//}
```

Comment:
This should either be removed or uncommented IMO

Reply:
It's for future contributors, Callinou asked for errors but none of the other property hints error, so incase it changes in the future, this would just need to be uncommented

Reply:
Should probably have a note to that effect

Reply:
You can potentially slap a TODO comment if this is not strictly necessary to have it merged now.
```suggestion
	// TODO: Show an error when an invalid hint is entered.
```

Reply:
@Mickeon the whole point of the comment is so that when others implement errors for the other hints, they can just uncomment this and it will be done, but if you think it's too useless or something, then i can just remove it

Reply:
@Dynamic-Pistol  I'd say remove it.

---

For what I gather:

1. That is a desired feature, but it is beyond the scope of this one.
2. You do not want to lose it.

You say it is for future contributors, but if it is done it is done, you are the future contributor. *In fact, if somebody implemented it by just uncommenting it, you'd deserve - at least - co-authorship.*

So, this is my suggestion: you can make a new PR for it, say in the description that it depends on this one. And you can point to the comments where it was discussed to justify it (assuming there is no proposal for it).

Unless you really don't care about it. In that case, just remove it. If or when there is a proposal, the code can be salvaged from this conversation. *In a sense, it is already a comment, since it is here in github*, so you don't need to keep in the code. *Anyway, if there is no proposal, I think it would be better off with one, that way it would make easier to find it and track it.*